Az.DevTestLabs2.psm1

```diff
@@ -260,13 +266,20 @@ function DeployLab {
       $lab = Get-AzureRmResource -Name $Name -ResourceGroupName $ResourceGroupName -ExpandProperties
     }
 
+    if ($VmCreationResourceGroupName) {
+
+      Set-AzDtlVmCreationResourceGroup -Lab $lab -VmCreationResourceGroupName $VmCreationResourceGroupName
```

Comment:
This call returns a lab correct?  If so, we can just capture it here (instead of making the extra call on the next line)...  Otherwise it will end up on the pipeline accidently